xtask/src/main.rs

```diff
@@ -89,8 +89,10 @@ fn run_action(workspace: Workspace, target: &str, extra_args: &[String]) {
                 let tracing_kind = find_tracing_kind(&rust_flags);
                 rust_flags = make_internal_rustflags(&rust_flags);
 
-                // Optimise the internal workspace.
-                cmd.arg("--release");
+                if target != "bench" {
+                    // Optimise the internal workspace.
+                    cmd.arg("--release");
+                }
```

Comment:
Why is this necessary? Shouldn't the internal workspace always be optimized?

Reply:
```
$ cargo bench --release
error: Found argument '--release' which wasn't expected, or isn't valid in this context
```

I guess the implication is that benchmarks are always optimised.

Reply:
Can you add a comment?

Reply:
I think we can just tweak the existing comment to something like "`--bench` implies `--release`; otherwise we need to add it"?

Reply:
Try f1eb1c9